modules/opportunities/client/controllers/opportunities.client.controller.js

```diff
@@ -37,12 +37,13 @@
 			'opportunity',
 			'Authentication',
 			'OpportunitiesService',
+			'ProposalsService',
 			'Notification',
 			'modalService',
 			'ask',
 			'myproposal',
 			'OpportunitiesCommon',
-			function($state, $stateParams, $sce, $location, opportunity, Authentication, OpportunitiesService, Notification, modalService, ask, myproposal, OpportunitiesCommon) {
+			function($state, $stateParams, $sce, $location, opportunity, Authentication, OpportunitiesService, ProposalsService, Notification, modalService, ask, myproposal, OpportunitiesCommon) {
```

Comment:
Do we need to include "OpportunitiesService" here, or can we remove it?

Reply:
Yes